packages/forma-36-website/src/components/MDXPage.js

```diff
@@ -150,57 +110,86 @@ export default function MDXPage({ mdxContent, frontmatter, propsMetadata }) {
-                  >
-                    <img src={githubIcon} alt="" />
-                    <span>Github</span>
-                  </a>
+              <div className={styles.subheaderRow}>
+                <div className={styles.buttonList}>
+                  {frontmatter?.storybook && (
+                    <a
+                      className={styles.imageLink}
+                      href={frontmatter.storybook}
+                      title={`View ${frontmatter?.title} in Storybook`}
+                    >
+                      <img src={storybookIcon} alt="" />
+                      <span>Storybook</span>
+                    </a>
+                  )}
+                  {frontmatter?.github && (
+                    <a
+                      className={styles.imageLink}
+                      href={frontmatter.github}
+                      title={`View ${frontmatter?.title} on GitHub`}
+                    >
+                      <img src={githubIcon} alt="" />
+                      <span>Github</span>
+                    </a>
+                  )}
+                </div>
+
+                {frontmatter?.status && (
+                  <span className={styles.badge}>
+                    <Badge
+                      variant={
+                        frontmatter.status === 'alpha'
+                          ? 'warning'
+                          : frontmatter?.status === 'deprecated'
+                          ? 'negative'
+                          : 'positive'
+                      }
+                    >
+                      {frontmatter.status}
+                    </Badge>
+                  </span>
                 )}
               </div>
-
-              {frontmatter?.status && (
-                <span className={styles.badge}>
-                  <Badge
-                    variant={
-                      frontmatter.status === 'alpha'
-                        ? 'warning'
-                        : frontmatter?.status === 'deprecated'
-                        ? 'negative'
-                        : 'positive'
-                    }
-                  >
-                    {frontmatter.status}
-                  </Badge>
-                </span>
-              )}
-            </div>
-          </header>
+            </header>
+          )}
 
           <main>
+            {frontmatter.v4Doc && (
+              <DisplayText as="h1" size="large">
+                {frontmatter.title}
+              </DisplayText>
+            )}
+
             <MDXRenderer>{mdxContent}</MDXRenderer>
+
+            {frontmatter.v4Doc && (
+              <>
+                <Heading
+                  as="h2"
+                  id="help-improve-this-page"
+                  marginTop="spacing2Xl"
+                >
+                  Help improve this page
+                </Heading>
+                <Stack>
+                  <TextLink href={frontmatter.github} target="_blank">
```

Comment:
Seems like this link does not work for some reason on deployed preview, not sure why.

Reply:
I've just tested and it opened this page: https://github.com/contentful/forma-36/tree/forma-v4/packages/components/accordion

Maybe you need to clean your cache

btw, I think all Github links are broken in production